woocommerce/storefront
# Touch devices dropdown support

Adds touch support for navigation dropdowns when using an handheld device such as an iPad.

How to test:

1. Create a menu with dropdowns and assign it to the "Primary navigation" area.

2. Use an iPad or switch Chrome to mobile mode and select the iPad Pro from the list of devices (if using Chrome, reload the page after switching to the iPad). 

3. Tap on the drop down to open it. The first tap should open the dropdown. A second tap on the same item should navigate away to the correct page.

Closes #962.
